prov/shm/src/smr_signal.h

```diff
@@ -62,8 +62,9 @@ static void smr_handle_signal(int signum, siginfo_t *info, void *ucontext)
 	/* call the original handler */
 	if (old_action[signum].sa_flags & SA_SIGINFO)
 		old_action[signum].sa_sigaction(signum, info, ucontext);
-	else if (old_action[signum].sa_handler == SIG_DFL ||
-		 old_action[signum].sa_handler == SIG_IGN)
+	else if (old_action[signum].sa_handler == SIG_DFL)
```

Comment:
We were raising the original signal for all intercepted ones at some point which would have passed the default signal down to the OS, but that seems to have changed in https://github.com/ofiwg/libfabric/commit/e663fdbeae855db1543354796e2c9d90ee142588#. 

This change itself makes sense to me, but please expand on the commit message like Sean pointed out. Which signal /app did you notice this issue with?